src/leetCodeExplorer.ts

```diff
@@ -64,57 +73,100 @@ export class LeetCodeTreeDataProvider implements vscode.TreeDataProvider<LeetCod
         }
 
         const idPrefix: number = Date.now();
-        return {
+        return { // element.id -> parent node name, element.name -> node name
```

Comment:
Why we save parent node name into `element.id`?

Reply:
There should be some field to store the parent node name when it is not problem node...since in previous commits you simply set `node.id` and `node.name` the same value, I decided to reuse the `id` field to represent some other value.

The reasons for the necessity of parent node name are:
1. We should be able to retrieve data from the `LeetCodeTreeDataProvider.treeData`.
2. One problem will now be added as `LeetCodeNode` at least 3 times(in `Difficulty`, `Tag` and `Company`), we should be able to distinguish them, hence the following naming rule:
```typescript
id: `${idPrefix}.${element.id}.${element.name}`,
```
(`idPrefix = Date.now()` just returns same value throughout initialization process)

Reply:
Maybe we could add a filed in the `LeetCodeNode` called 'parentName' to store the parent node info.